Clarifying recording flow in case of config options (#11619)
DataDog/documentation: content/en/synthetics/browser_tests/_index.md

  * **Cookies**: Define cookies to add to the default browser cookies. Set multiple cookies using the format `<COOKIE_NAME1>=<COOKIE_VALUE1>; <COOKIE_NAME2>=<COOKIE_VALUE2>`.
  * **Proxy URL**: URL of the proxy the requests should go through (`http://<YOUR_USER>:<YOUR_PWD>@<YOUR_IP>:<YOUR_PORT>`).
  

> **Note:** These options are set at every test execution and on every step of your browser test. They are however only applied at execution time and not at recording time. You should consequently manually apply these on the page that is being used for recording and only then record steps the test should go through.

Author: ```suggestion
**Note:** These options are set at every test execution and for every step of your browser test. They are however only applied at execution time and not at recording time. You should consequently manually apply these on the page that is being used for recording and only then record steps the test should go through.
```

> **Note:** These options are set at every test execution and for every step of your browser test. They are however only applied at execution time and not at recording time. You should consequently manually apply these on the page that is being used for recording and only then record steps the test should go through.

Reviewer: ```suggestion
**Note:** These request options are set at every test execution and apply to every step of your browser test at execution time, not recording time. You can manually apply these options on the page you are recording and selectively record those steps for your test.
```
